get-started/02_our_app.md

```diff
@@ -0,0 +1,118 @@
+---
+title: "Our Application"
+keywords: get started, setup, orientation, quickstart, intro, concepts, containers, docker desktop
+description: overview of our simple applicaiton for learning docker
+---
+
+
+For the rest of this tutorial, we will be working with a simple todo
+list manager that is running in Node.js. If you're not familiar with Node.js,
+don't worry! No real JavaScript experience is needed!
+
+At this point, your development team is quite small and you're simply
+building an app to prove out your MVP (minimum viable product). You want
+to show how it works and what it's capable of doing without needing to
+think about how it will work for a large team, multiple developers, etc.
+
+![Todo List Manager Screenshot](images/todo-list-sample.png){: style="width:50%;" }
+
+
+## Getting our App
+
+Before we can run the application, we need to get the application source code onto 
+our machine. For real projects, you will typically clone the repo. But, for this tutorial,
+we have created a ZIP file containing the application.
+
+1. [Download the ZIP](/assets/app.zip). Open the ZIP file and make sure you extract the
```

Comment:
The link gives an error

Reply:
Pointed to the repo that has this actual code in.